test-java-lib/src/main/java/com/uber/lib/unannotated/AndroidRecentlyAnnotatedClass.java

```diff
@@ -0,0 +1,51 @@
+package com.uber.lib.unannotated;
+
+import androidx.annotation.RecentlyNonNull;
+import androidx.annotation.RecentlyNullable;
+
+// don't actually check this code
+@SuppressWarnings("NullAway")
```

Comment:
Do we need this? Why aren't we excluding test code from checking? Also, `unannotated` shouldn't be checked inside the tests, should it?

Reply:
Without this, when `test-java-lib` gets compiled as part of the standard Gradle build we see a bunch of NullAway errors 😄 We could disable NullAway on all of the `test-java-lib` library I guess.

Reply:
I think we should disable NullAway for all the `test-` projects, since this is going to be a constant issue, plus we don't want to suppress things when actually analyzing them in tests, do we? Could be convinced of the opposite approach, but this just feels weird. At least I would add `com.uber.lib.unannotated` to excluded sub-packages in our gradle config. Maybe that's the better approach?

Reply:
Yup I added `com.uber.lib.unannotated` to unannotated sub-packages